kalman_watch/monitor_win_perigee.py

```diff
@@ -529,41 +535,100 @@ def _reshape_to_n_sample_2d(arr: np.ndarray, n_sample: int = 60) -> np.ndarray:
     return arr
 
 
-def get_binned_drops_from_npnt_tlm(
-    dat: fetch.MSID, n_sample: int = 60
+def get_binned_drops_from_event_perigee(
+    ep: EventPerigee,
 ) -> tuple[np.ndarray, np.ndarray]:
-    """Get the number of kalman drops per minute from NPNT telemetry.
+    """Get the number of kalman drops per "minute" from NPNT telemetry.
```

Comment:
I think the docstrings need update if this is now getting the fraction of samples with IR flag instead of kalman drops.